client/components/MainDescription.jsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
 
-const MainDescription = ({ description }) => {
-  return <h2>Hey here it is!</h2>;
-};
+class MainDescription extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showMore: false,
+    };
+    // this.showMore = this.showMore.bind(this);
+  }
+
+  showMore(event) {
+    this.setState({ showMore: true });
+  }
```

Comment:
should showMore be a method be more modular in the sense that it just changes showMore to the opposite of its current state?  that way you can use it to hide the "more" observations as well?  trying to remember if the site even does that and can't find it on my example page.

Reply:
The way its implemented on Airbnb is that once you click "read more" you cannot then hide - no "read less" functionality exists. It's a one time thing. Thats a good question though!